test/test_formatters.rb

```diff
@@ -100,6 +100,15 @@ def test_format_value_does_not_modify_resource_identifiers
     identifier = '@android:string/cancel'
     assert_equal identifier, @formatter.format_value(identifier)
   end
+
+  def test_deducts_language_from_resource_folder
+    language = %w(en de fr).sample
```

Comment:
It might be good to test one of the more complicated language codes, here. Like `zh-rCN`